contracts/sNOTE.sol

```diff
@@ -569,7 +575,7 @@ contract sNOTE is
 
         // NOTE: at this point the BPT has already been transferred into the sNOTE contract, so this
         // bptBalance amount includes bptAmount.
-        uint256 bptBalance = BALANCER_POOL_TOKEN.balanceOf(address(this));
+        uint256 bptBalance = LIQUIDITY_GAUGE.balanceOf(address(this));
```

Comment:
To be safe here, can we make an internal method that adds together the balances on both the POOL TOKEN and the LIQUIDITY GAUGE and then use that wherever we need to query the balanceOf